wallet/src/de/schildbach/wallet/ui/preference/ResolveDnsTask.java

```diff
@@ -1,66 +0,0 @@
-/*
- * Copyright 2016 the original author or authors.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package de.schildbach.wallet.ui.preference;
-
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-
-import android.os.Handler;
-import android.os.Looper;
-
-/**
- * @author Andreas Schildbach
- */
-public abstract class ResolveDnsTask {
-    private final Handler backgroundHandler;
-    private final Handler callbackHandler;
-
-    public ResolveDnsTask(final Handler backgroundHandler) {
-        this.backgroundHandler = backgroundHandler;
-        this.callbackHandler = new Handler(Looper.myLooper());
-    }
```

Comment:
This may have been used for the Trusted Peer feature which didn't make from version 6 to version 7.

It is a feature that I would like to bring back, but we can always go back in time to get the code and probably rewrite most of it.

Reply:
I agree. If there is a story for bringing it back, we can mention this file in there.